ipaplatform/base/paths.py

```diff
@@ -378,6 +378,6 @@ class BasePathNamespace(object):
     KEYCTL = '/usr/bin/keyctl'
     GETENT = '/usr/bin/getent'
     SSHD = '/usr/sbin/sshd'
-
+    SSSCTL = '/usr/sbin/sssctl'
```

Comment:
You have to keep the two new lines between the last constant and the module level variables. One line is a PEP 8 violation.